integration-test/tests/cms/codeofconduct.test.ts

```diff
@@ -0,0 +1,12 @@
+import {expect, test} from '@playwright/test';
+import { codeofconductExepctedInformation } from '../../utils/datafactory/codeofconduct.data';
+test('GET /api/cms/v1/code-of-conduct returns correct data', async ({request}) => {
+    const response = await request.get(`/api/cms/v1/code-of-conduct`);
+ 
+    expect(response.status()).toBe(200);
+    const body = await response.json();
```

Comment:
Before checking response, it is important to include JSON schema validation. Please, see the example in ´footer.test.ts´ test file.

Reply:
@nora-weisser  Thanks , Schema validation in ´footer.test.ts´ test file. merged ? as I cannot see the example in my test file ? Though I can check in GitHub commits.
![image](https://github.com/user-attachments/assets/f7d037c8-57f6-405a-9db3-75e93072b4b0)

Reply:
We discussed that this will be done in another PR